Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMeshPrimitiveAttributes.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// Common mesh primitive attributes.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/mesh.primitive.schema.json
+    /// </summary>
+    /// <remarks>
+    /// Application specific semantics are not supported
+    /// </remarks>
+    [Serializable]
+    public class GltfMeshPrimitiveAttributes
+    {
+        public int POSITION = -1;
```

Comment:
Should update capitalization (ex: POSITION -> Position).

Reply:
Again, this matches the schema per the json serializer constraints. They need to match the casing or it will fail.

Reply:
synced offline, everyones onboard with this naming.